### PR TITLE
vpc and instances created

### PR DESCRIPTION
**Pull request by divesh987**
Vpc created 
RDS instance created 
App instance created

### CODE REVIEW

File: Berksfile.lock [hunk 1]
```diff
@@ -0,0 +1,36 @@
+DEPENDENCIES
```
Comment (steveyblam) [inline Berksfile.lock]: Please remove this file and add to ignore

File: log/development.log [hunk 2]
```diff
@@ -11993,3 +11993,168 @@ Processing by LogsController#index as HTML
 Completed 200 OK in 691ms (Views: 162.4ms | ActiveRecord: 1.9ms)
```
Comment (steveyblam) [inline log/development.log]: Please remove this file and add to gitignore

File: terraform/main.tf [hunk 3]
```diff
@@ -0,0 +1,37 @@
+provider "aws" {
+  region  = "eu-west-2"
+}
+
+resource "aws_vpc" "inventory-vpc" {
+  tags {
+      Name = "Inventory - VPC"
+  }
+  cidr_block = "11.3.0.0/16"
+
```
Comment (steveyblam) [inline terraform/main.tf]: remove this white space

File: terraform/main.tf [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+
+resource "aws_db_instance" "inventory-db" {
+  engine               = "postgres"
+  instance_class =  "db.t2.micro"
+  allocated_storage = 8
+  name = "spartaInventoryDb"
+  username ="inventory"
+  password = "sparta123"
```
Comment (steveyblam) [inline terraform/main.tf]: Please generate a random string for this password so it is not stored in git.

https://www.terraform.io/docs/providers/random/r/string.html

**Comment by divesh987** [comment]
Changes made as requested

**Pull request merged**